data/text/battle.asm

```diff
@@ -881,7 +881,7 @@ ReflectEffectText:
 	line "DEFENSE rose!"
 	prompt
 
-NothingHappenedText:
+SquirtbottleNothingText:
```

Comment:
This is also used by `PrintNothingHappened` for Sketch and Splash, so `NothingHappenedText` was appropriate.